CI/check-jsonschema.py

```diff
@@ -0,0 +1,45 @@
+import json
```

Comment:
Maybe that's just me, but I was taught (a lot of moons ago) to wrap code _at the very least_ in a `main` function and run with:

```
if __name__ == '__main__':
    sys.exit(main())
```

Also there are not checks whether the file specified by `sys.argv[1]` exists, so at the very least the `open` calls should be wrapped in a `try..except..finally` block.